docs/source/commands/init.mdx

```diff
@@ -109,8 +112,90 @@ Once you've completed the wizard, it provides a [`rover dev`](./dev) command wit
+
+
+## Initializing an MCP server
+### `rover init --mcp`
+
+Rover's `init --mcp` command provides a helpful starting point for working with the [Apollo MCP Server](https://www.apollographql.com/docs/apollo-mcp-server). 
+
+```terminal showLineNumbers=false
+rover init --mcp
+```
+
+The `init` CLI wizard walks you through the process of configuring and creating an MCP server, with MCP tools based on your GraphQL API operations.
+
+The command creates the following files:
+
+- MCP config YAML for local and staging scenarios
+- MCP config for adding to Claude Desktop
+- Starter MCP tools, based on your GraphQL API operations
+- `.env` files for running the GraphQL API and MCP Server
+- A set of local configuration files and boilerplate code that represent the graph
+- mcp.Dockerfile (optional Docker container) 
+
+##### Options
+
+You can also pass options directly to `rover init --mcp` instead of answering prompts in the wizard. The wizard still asks for any information not provided via an option and always asks for confirmation before creating any files locally.
+
+```terminal showLineNumbers=false
+rover init --mcp \
+  --project-type <PROJECT_TYPE> \
+  --org-id <ORGANIZATION_ID> \
+  --project-name <PROJECT_NAME> \
+  --graph-id <GRAPH_ID>
+```
+
+##### Available options
+
+| Option | Description | Possible Values |
+| ------ | ----------- | --------------- |
+| `--project-type` | Whether to create MCP tools from a new Apollo GraphOS project or an existing Apollo GraphOS project. | `create-new`, <br/>`add-subgraph` |
+| `--org-id` | The ID of the [GraphOS organization](/graphos/platform/access-management/org#view-your-organizations) where the graph should be created. | *(your organization ID)* |
+| `--project-name` | The name for your new graph. | *(your graph name)* |
+| `--graph-id` | The ID for your graph in GraphOS. (This must be a unique identifier.) | *(your graph ID)* |
+
+
+### Next Steps
+
+Once you've completed the wizard, you will be shown recommended next steps for using and running your MCP server.
+
+
+#### Connect your MCP server to Claude Desktop
+
+```bash
+# Copy the configuration for Claude Desktop
+# macOS:
+cp claude_desktop_config.json ~/Library/Application\ Support/Claude/claude_desktop_config.json
+
+# Windows:
+copy claude_desktop_config.json "%APPDATA%\Claude\claude_desktop_config.json"
+
+# Linux:
+cp claude_desktop_config.json ~/.config/Claude/claude_desktop_config.json
+
+# Restart Claude Desktop
+```
+
+For instructions on adding your MCP server to other AI clients, see the [MCP Quickstart guide](https://www.apollographql.com/docs/apollo-mcp-server/quickstart).
+
+
+#### Run your MCP server and GraphQL API
+
+When the `rover init` wizard is complete, you can use [`rover dev`](./dev) to start everything together:
+
```

Comment:
```suggestion
```
This becomes redundant when we move up this section

Reply:
Actually I would remove this section and just link to the `rover dev` command to keep this article tight (which is really about the `init` command)